doc/source/contributing.rst

```diff
@@ -170,7 +170,7 @@ We'll now kick off a three-step process:
 
    # Create and activate the build environment
    conda env create -f ci/environment-dev.yaml
-   conda activate pandas-dev
+   source activate pandas-dev
```

Comment:
note that ``conda activate`` is not actually a typo, you just do this in newer versions of conda. can you update so both are here? (its ok that source activate is the primary).

Reply:
I stand corrected-- looks like `conda activate` was added around version 4.4.